src/libs/actions/Report.js

```diff
@@ -123,6 +123,27 @@ function getNormalizedTypingStatus(typingStatus) {
     return normalizedTypingStatus;
 }
 
+/**
+ * There are 2 possibilities that we can receive via pusher for a user's leaving status:
+ * 1. The "new" way from New Expensify is passed as {[login]: Boolean} (e.g. {yuwen@expensify.com: true}), where the value
+ * is whether the user with that login is leaving on the report or not.
+ * 2. The "old" way from e.com which is passed as {userLogin: login} (e.g. {userLogin: bstites@expensify.com})
+ *
+ * This method makes sure that no matter which we get, we return the "new" format
+ *
+ * @param {Object} leavingStatus
+ * @returns {Object}
+ */
+function getNormalizedLeavingStatus(leavingStatus) {
```

Comment:
Can we just rename the existing method to something like `getNormalizedStatus`? There's no need to duplicate that logic

Reply:
@luacmartins did you mean to make typing and leaving the same method?

```
function getNormalizedStatus(typingStatus) {
    let normalizedStatus = typingStatus;

    if (_.first(_.keys(typingStatus)) === 'userLogin') {
        normalizedStatus = {[typingStatus.userLogin]: true};
    }

    return normalizedStatus;
}
```

Reply:
yup, you got it right!